examples/tts/conf/fastpitch.yaml

```diff
@@ -0,0 +1,150 @@
+name: &name "FastPitch"
+sample_rate: &sr 22050
+n_fft: &n_fft 1024
+n_mels: &n_mels 80
+fmax: &fmax 8000
```

Comment:
Remove the yaml anchors

Reply:
The anchors are used later. Do you want to duplicate the values without anchoring?
btw These were taken from [glow_tts.yaml](https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/glow_tts.yaml).

Reply:
Yes, the TTS yamls are in a bad place and I need to clean them up. You can use hydra/omegaconf interpolation instead like here: https://github.com/NVIDIA/NeMo/pull/1799/files/e137574cbe6348159a8b4366afc9a13fe013a805#diff-c7f54a968edf3ae2c9d4b432a14edea0820b645004457eb02ad5cca04603683bR19

Reply:
Done